src/main/java/com/acrolinx/client/demo/DemoUtil.java

```diff
@@ -109,7 +106,7 @@ private static GuidanceProfile getGuidanceProfiles(
    * @see <a href="https://github.com/acrolinx/platform-api#authentication">Acrolinx Platform
    *     API</a>
    * @see <a href=
-   *     "https://github.com/acrolinx/acrolinx-coding-guidance/blob/main/topics/configuration.md#authentication">Authentication
+   *     <p>"https://github.com/acrolinx/acrolinx-coding-guidance/blob/main/topics/configuration.md#authentication">Authentication
```

Comment:
where did this "<p>" come from? Anyway, must have been a mistake, should be removed again.

Reply:
done